test/integration/generate.sh

```diff
@@ -10,6 +10,6 @@ metadata:
   namespace: openshift-machine-api
 type: Opaque
 data:
-  awsAccessKeyId: $(echo -n $(aws configure get aws_access_key_id) | base64)
-  awsSecretAccessKey: $(echo -n $(aws configure get aws_secret_access_key) | base64)
+  aws_access_key_id: $(echo -n $(aws configure get aws_access_key_id) | base64)
+  aws_secret_access_key: $(echo -n $(aws configure get aws_secret_access_key) | base64)
```

Comment:
can you reference the counterpart PR why this i needed?

Reply:
https://github.com/openshift/cluster-api-provider-aws/pull/164